### ISSUE TITLE
Toggling ERB Commands Doesn't Work In Latest Dev Build

### ISSUE THREAD

**Issue by eddorre** [eddorre/SublimeERB#6]
Just opening a ticket for myself so that I can track it. Noticed that the latest dev build (2170) of Sublime Text 2 breaks the toggling feature of SublimeERB.

**Comment by eddorre** [comment]
Further documentation:

Bug reported here: http://sublimetext.userecho.com/topic/93699-command_history-does-not-help-determine-last-command/

It looks like the API for the command history has
changed a bit.

In the stable release of Sublime Text, when running my command, the
command history (self.view.command_history) is listed as such:

(None, None, 0)
COMMAND HISTORY
(u'erb', None, 1)
COMMAND HISTORY
(u'erb', None, 1)
COMMAND HISTORY
(u'erb', None, 1)
COMMAND HISTORY
(u'erb', None, 1)
COMMAND HISTORY
(u'erb', None, 1)
COMMAND HISTORY
(u'erb', None, 1)
COMMAND HISTORY
(u'erb', None, 1)
COMMAND HISTORY
(u'erb', None, 1)

This allows me to find out of they have not done anything else in between key presses which allows me to toggle ERB
blocks.

In the latest dev build, the command history looks something like this:

COMMAND HISTORY
(None, None, 0)
COMMAND HISTORY
(None, None, 0)
COMMAND HISTORY
(u'erb', None, 2)
COMMAND HISTORY
(u'erb', None, 2)
COMMAND HISTORY
(u'left_delete', None, 1)
COMMAND HISTORY
(u'left_delete', None, 1)
COMMAND HISTORY
(u'left_delete', None, 1)
COMMAND HISTORY
(u'erb', None, 3)
COMMAND HISTORY
(u'erb', None, 3)
COMMAND HISTORY
(u'erb', None, 3)
COMMAND HISTORY
(u'erb', None, 3)
COMMAND HISTORY
(u'erb', None, 3)
COMMAND HISTORY
(u'erb', None, 3)
COMMAND HISTORY
(u'erb', None, 6)
COMMAND HISTORY
(u'erb', None, 6)

**Comment by eddorre** [comment]
Well, despite emailing the author of Sublime Text 2 directly and opening bug reports, the change to the command_history API call changed and there appears to be no other replacement. That's frustrating.

Now that the latest update is considered to be in the stable build, I'll have to look at alternatives when toggling ERB blocks.

**Comment by jopotts** [comment]
Thanks for the excellent plugin. Looking forward to the fix.

**Comment by edubkendo** [comment]
ah that explains the problem i suddenly started having. damn. I was hoping that it was something caused by my switch to ubuntu and that there would be a quick fix. Thanks a ton for such a great plug-in. I use (used?) it daily. Well, it should still be helpful for getting the initial tags in place. Will just have to manipulate them manually for now. (The most irritating part is that I had a habit of, whenever I was stuck and mulling over how to do something, of sitting there and toggling through the erb tags while thinking. Totally disrupted my day when the first time I started to do that I got a long string of tags.)
